api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -493,4 +515,8 @@ public FirecloudNihStatus getNihStatus() {
           }
         });
   }
+
+  private boolean isFireCloudBillingV2ApiEnabled() {
```

Comment:
nit: I don't think the code reuse here really adds that much in LOC saved or easier readability. The chained object call is just as clear IMO and if anything, I thought (incorrectly) that there was something more going on when I read the function above because it was factored out.

Reply:
I'm a big fan of small one time use private helper methods in general, especially when its not obvious what the one liner does. In this case, I felt like the one liner implementation was straight forward enough that it doesn't actually save me any thinking time in deciphering what it does.

Just a nit though and giving my opinion. I think the helper function is fine and you can keep it in.